mail_tracking/models/mail_alias.py

```diff
@@ -10,9 +10,15 @@ class MailAlias(models.Model):
     @api.model
     @tools.ormcache()
     def get_aliases(self):
-        return set(x['display_name'] for x in self.search_read([
+        aliases = set(x['display_name'] for x in self.search_read([
             ('alias_name', '!=', False),
         ], ['display_name']))
+        IrConfigParamObj = self.env["ir.config_parameter"].sudo()
+        catchall = "%s@%s" % (
```

Comment:
The rest of the aliases don't contain the domain. Why this one does?

Reply:
With mail.alias records we are using "display_name" that includes the catchall domain.